### ISSUE TITLE
Add links to help pages in requests

### ISSUE THREAD

**Issue by aapeliv** [Couchers-org/couchers#4392]
* On the host side: when you receive a request, there should be a helpful little text near the accept/reject buttons directing the user to a help page on how to respond to a request. "Not sure how to respond? Have a look at our guide here." or similar
* On the surfer side: there should be some kind of help text when writing a request "Read our guide to writing a great request" or similar
* On the surfer side, if they get rejected, there should be a link saying "Need help having your requests accepted? Read our guide here for the best tips!"

**Comment by nabramow** [comment]
@aapeliv what's this blocked by?

**Comment by aapeliv** [comment]
> @aapeliv what's this blocked by?

It's waiting on @jesseallhands to finish up the guides in the help center!

**Comment by jesseallhands** [comment]
> It's waiting on @jesseallhands to finish up the guides in the help center!

They are done...

### 1. Helper text for recipient of a request:

> [Things to consider](https://help.couchers.org/hc/couchersorg-help-center/articles/1715125658-what-are-some-things-i-should-think-about-before-responding-to-a-request) before responding.

Put the helper text here above the accept or decline buttons with some kind of dividing line above the text (to separate it from the messages):

![image](https://github.com/user-attachments/assets/e3fd47d3-92d7-4095-a475-62ad94994b77)

### 2. Helper text for senders of a request:

> [Read our guide](https://help.couchers.org/hc/couchersorg-help-center/articles/1725943310-quick-reference-writing-great-requests) on how to write a request that will get accepted.

The link takes you to an (objective) quick guide which also provides a link to a more detailed (subjective) guide with examples.

The text should appear on the desktop request page like this:

![image](https://github.com/user-attachments/assets/8a32bb8d-eeb0-4b6e-a7c6-bdbdd7b31498)

### 3. Helper text for rejected requests can use the same text as Nr. 2 above, but instead of putting the helper text above the buttons, put them below.

I moved this issue to ready since everything needed should be available now.

**Comment by aapeliv** [comment]
I meant on the help center. These still have TODOs:

https://chatwoot.couchershq.org/app/accounts/2/portals/couchersorg-help-center/en/articles/56

https://chatwoot.couchershq.org/app/accounts/2/portals/couchersorg-help-center/en/articles/55

**Comment by jesseallhands** [comment]
> I meant on the help center. These still have TODOs:
> 
> https://chatwoot.couchershq.org/app/accounts/2/portals/couchersorg-help-center/en/articles/56

The responding to requests one is covered here, it's just on a different help page article/different title (I think i had created this before you started creating yours): "[Things to consider](https://help.couchers.org/hc/couchersorg-help-center/articles/1715125658-what-are-some-things-i-should-think-about-before-responding-to-a-request) before responding."

> https://chatwoot.couchershq.org/app/accounts/2/portals/couchersorg-help-center/en/articles/55

The profile help topic is one we should add, but it is not mentioned in this issue. It needs its own issue.

**Comment by aapeliv** [comment]
Ah okay! My bad. So the help pages are all ready to go?

**Comment by jesseallhands** [comment]
I'll add the part about giving feedback and responding to requests to the "Things to consider" page though!

**Comment by jesseallhands** [comment]
> Ah okay! My bad. So the help pages are all ready to go?

Yes, good to go. I moved this issue to ready.
